libnavigation-base/src/main/res/values-uz/strings.xml

```diff
@@ -2,7 +2,8 @@
 
     <!-- Time formatting -->
     <plurals name="mapbox_number_of_days">
-        <item quantity="other">kun</item>
+        <item quantity="one">kun</item>
+        <item quantity="other">kunlar</item>
```

Comment:
Are you sure about this? Both Yandex and Google translate "twenty days" as "yigirma **kun**".

Reply:
thanks, let me double check it 👍

Reply:
great catch! "days" is translated like "kunlar" but "20 days" is "20 kun". It's better not to assume that other languages works like the English and translate in the context 😅